src/main/resources/db/migration/V2024_10_16_1200__CIV-15081_cosc_generate_certificate_defendant.sql

```diff
@@ -0,0 +1,16 @@
+/**
+ * Add scenario
+ */
+INSERT INTO dbs.scenario (name, notifications_to_delete, notifications_to_create)
+VALUES ('Scenario.AAA6.ProofofDebtPayment.ApplicationProcessed.Defendant',
+        '{"Notice.AAA6.ProofofDebtPayment.Application.Defendant", "Notice.AAA6.ProofofDebtPayment.Application.Claimant"}',
+        '{"Notice.AAA6.ProofofDebtPayment.ApplicationProcessed.Defendant":[]}');
+
+INSERT INTO dbs.dashboard_notifications_templates (template_name, title_En, title_Cy, description_En, description_Cy
+                                                  ,notification_role)
+VALUES ('Notice.AAA6.ProofofDebtPayment.ApplicationProcessed.Defendant',
+        'Proof of debt payment issued',
+        'Cadarnhad o daliad dyled dyfarniad',
+        '<p class="govuk-body">You’ve successfully confirmed that you’ve paid a judgment (CCJ) debt in full. The public register will be updated and you can now <a href="{CONFIRM_YOU_HAVE_BEEN_PAID_URL}" class="govuk-link">view the certificate</a>. You should download this certificate for your records. The certificate can be found in ‘Orders and notices from the court’.</p>',
```

Comment:
{CONFIRM_YOU_HAVE_BEEN_PAID_URL} (only for testing)
this url would be changed to {VIEW_COSC_CERTIFICATE_URL}